recipes/oci/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "oci" %}
+{% set version = "2.88.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/oci-{{ version }}.tar.gz
+  sha256: c7c1ad5468417b23349487796a2b6145379c06f829e21ca445b6ae81673058bb
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
```

Upstream [explicitly says ](https://github.com/oracle/oci-python-sdk/tree/v2.88.0#about) 3.6 to 3.9, so let's at least start with 3.6:
```suggestion
    - python >=3.6
    - pip
  run:
    - python >=3.6
```